teams/wg-rls-2.toml

```diff
@@ -4,10 +4,13 @@ kind = "working-group"
 
 [people]
 leads = ["matklad"]
-members = ["matklad"]
+members = ["matklad", "lnicola", "Veykril", "flodiebold", "jonas-schievink"]
 
 [website]
 name = "RLS 2.0 working group"
 description = "Experimenting with a new compiler architecture tailored for IDEs"
 repo = "https://rust-lang.github.io/compiler-team/working-groups/rls-2.0/"
 zulip-stream = "t-compiler/wg-rls-2.0"
```

Comment:
I just realized I think we should update this? The zulip stream is at `t-compiler/rust-analyzer` now

Reply:
Yeah, updated it